File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio16 ultima cifra de un numero/archives/16.js
function sumarJS() {
  const a = parseInt(document.getElementById("num1").value);
  if (Number.isFinite(a)) {
    /* SECTION inicio codigo tras verificacion de formulario */
    let ultimo;
    if (a < 0) {
      ultimo = Math.abs(a) % 10;
    } else {
      ultimo = a % 10;
    }
    mensaje = `El ultimo digito  de ${a}:
                <br> es ${ultimo}`;
    /* !SECTION fin codigo tras verificacion de formulario */
  } else {
    mensaje = `Ingrese un número válido en el formulario `;
  }
  const resultadoDiv = document.getElementById("resultadoJavascript");
  resultadoDiv.innerHTML = `${mensaje}`;
}

File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio16 ultima cifra de un numero/archives/16.py
import sys
import math

# Obtener los argumentos de la línea de comandos
a = int(sys.argv[1])
ultimo = abs(a) % 10 if a < 0 else a % 10
mensaje = f"El ultimo digito de {a} <br>es: {ultimo}"
print(f"{mensaje}")
